src/Build.UnitTests/BackEnd/ResultsCache_Tests.cs

```diff
@@ -168,8 +168,8 @@ public void TestRetrieveSubsetResults()
 
         /// <summary>
         /// If a result had multiple targets associated with it and we only requested some of their 
-        /// results, the returned result should only contain the targets we asked for, BUT the overall 
-        /// status of the result should remain the same.  
```

Comment:
I hope this comment was wrong. As far as I can tell the idea behind `OverallResult` was exactly the opposite, i.e. correctly calculating results for subsets of targets.

Reply:
I'm not confident of this answer, but I think both versions of the comment are saying the same (correct) thing: if you only request some results, the Overall status reflects the results of all the targets, both those we asked for and those we didn't.